DSCResources/MSFT_xADGroupMember/MSFT_xADGroupMember.psm1

```diff
@@ -0,0 +1,140 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)]
+        [System.String]
+        $Identity,
+
+        [parameter(Mandatory = $true)]
+        [System.String]
+        $Members,
+
+        [parameter(Mandatory = $true)]
+        [ValidateSet("Present","Absent")]
+        [System.String]
+        $Ensure,
+
+        [System.Management.Automation.PSCredential]
+        $Credential
+    )
+
+    If($PSBoundParameters.ContainsKey('Credential'))
```

Comment:
It would be good to check if AD PowerShell module is present on the system or not and provide useful message to the user instead of the default 'command not found' error.